source/spelling_wordlist.txt

```diff
@@ -496,7 +496,7 @@ microarchitecture
 middleware
 minibatch
 minmag
-mkl
+onemath
```

Comment:
You're also adding `onemath` below line 554, one of them can be removed.
There is also an occurrence of `MKL` line 83 which can be removed, I don't think it should be used anymore.